src/schema/v2/sale/__tests__/index.test.js

```diff
@@ -22,6 +22,24 @@ describe("Sale type", () => {
     })
   }
 
+  describe("sale timeZone", () => {
+    const query = `
+    {
+      sale(id: "foo-foo") {
+        timeZone
+      }
+    }`
+
+    it("returns the correct value for time_zone", async () => {
+      sale.timeZone = "America/Chicago"
```

Comment:
The other examples I see in this file still show snake_case getters and setters but in the query, the query field is camelCased, which is a bit confusing.

Maybe @zephraph can clarify the way this should work.

Reply:
@bhoggard Is this resolved? I'm still curious if this should be `sale.time_zone` vs `sale.timeZone`. This file has some conflicting examples.

Reply:
We prefer camelcase (which is why _all_ fields are camelcased in metaphysics v2). We had started to migrate some fields already in v1, but are not consistent (as you're seeing here!).

Making it camelcased now means we'll have one less field to not change later when we migrate, but it's not a huge issue either way.

Reply:
Gotcha. Thanks! Sounds good :+1: